components/support/utils/src/main/java/mozilla/components/support/utils/DownloadUtils.kt

```diff
@@ -39,6 +39,14 @@ object DownloadUtils {
             "(?:;\\s*filename\\*\\s*=\\s*(utf-8|iso-8859-1)'[^']*'(\\S*))?",
             Pattern.CASE_INSENSITIVE)
 
+    /**
+     * Keys for the capture groups inside CONTENT_DISPOSITION_PATTERN
+     */
+    private val ENCODED_FILE_NAME_GROUP = 5
+    private val ENCODING_GROUP = 4
+    private val QUOTED_FILE_NAME_GROUP = 3
+    private val UNQUOTED_FILE_NAME = 2
+
```

Comment:
Let's do one more clean up and move the pattern as well as those groups into constants in the companion object:

```Kotlin
companion object {
   private const val ...
   private const val ...
   private const val ...
}
```